### DISCUSSION TITLE
Welcome to solaris Discussions!

### DISCUSSION THREAD

**Discussion by rbavery** [CosmiQ/solaris#437] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. The GeoAI team at Development Seed is taking on maintainership of this library and welcomes contribution, ideas, and discussion with the larger geospatial community. Check out this [roadmap](https://github.com/CosmiQ/solaris/blob/0.5.0/ROADMAP.md) which details our plan for v0.5.0 and beyond.
  
   We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, feel free to comment below with an introduction of yourself and tell us about what you do (or have done) with this tool!

**Comment by remtav** [comment]
Hey! My name is Remi Tavon. I'm a Geospatial Analyst at Natural Resources Canada. We're maintaining the [geo-deep-learning](https://github.com/NRCan/geo-deep-learning) project which aims at providing a pipeline for pre-processing, training and inferring geospatial data from any list of rasters and ground truth as polygons (so long as they are supported by, respectively, rasterio and geopandas). There is still a lot of overlap with the [torchgeo](https://github.com/microsoft/torchgeo) project which we follow closely. We will be integrating the torchgeo and solaris tools as much as possible in the coming months to slim down our project and rely mostly on external utilities. 

For us, the solaris project is very useful for the preprocessing step, especially to create patches from a raster scene, with accompanying ground truth. We also benefit from the polygon-based metrics for the evaluation semantic segmentation models on building footprints.

We would be more than happy to help out with the development of solaris. Seeing that the project is still active, I will definitely be opening issues and PRs to try and keep it up to date with our needs and those of the community.

**Reply by rbavery** [reply]
great to hear @remtav! I'm eager to hear more about how solaris is useful for the geo-deep-learning project and NRCAN. I'd also like to make sure we minimize overlap with torchgeo in solaris.
